src/tools/fuzzing/heap-types.cpp

```diff
@@ -42,8 +42,12 @@ struct HeapTypeGeneratorImpl {
   // allows us to choose HeapTypes that we know will be subtypes of data or func
   // before we actually generate the types.
   using BasicKind = HeapType::BasicHeapType;
-  struct SignatureKind {};
-  struct DataKind {};
+  struct SignatureKind {
+    SignatureKind() = default;
+  };
+  struct DataKind {
+    DataKind() = default;
+  };
```

Comment:
I am afraid to ask, but what weird C++ reason that I won't understand required this change?

Reply:
Oh, actually this is left over from when my compiler started emitting spurious warnings about unitialized memory. This was my attempt to fix it, but it didn't work. I ended up just switching compilers for the time being. Will revert this.